nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -241,6 +241,20 @@ in {
       '';
     };
 
+    rememberLastChoice = mkOption {
+      default = false;
+
+      type = types.bool;
+
+      description = lib.mdDoc ''
+        Remembers the last chosen systemd-boot entry. For example, given entries A and B. If B is
+	is selected, next boot B will automatically be highlighted instead of the latest generation.
```

Comment:
This should all be of consistent indentation so that it maintains proper formatting in the NixOS manual.